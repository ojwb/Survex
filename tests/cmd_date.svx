; pos=dump warn=23
; valid dates:
*fix _ 0 0 0
*begin
*date 00.01.01
_ 19000101 1 000 00
*date 00.01
_ 190001 2 000 00
*date 00.02
_ 190002 3 000 00
*date 00
_ 1900 4 000 00
*date 1900-2000
_ 1900_2000 5 000 00
*date 1900.01.02
_ 19000102 6 000 00
*date 1900.03
_ 190003 7 000 00
*date 1900.04
_ 190004 8 000 00
*date 1900.02.28
_ 19000228 9 000 00
*date 1901
_ 1901 10 000 00
*date 02.01.01
_ 19020101 11 000 00
*date 02.01
_ 190201 12 000 00
*date 02.02
_ 190202 13 000 00
*date 02
_ 1902 14 000 00
*date 1902-2000
_ 1902_2000 15 000 00
*date 1903.01.01
_ 19030101 16 000 00
*date 1903.01
_ 190301 17 000 00
*date 1903.02
_ 190302 18 000 00
*date 1903
_ 1903 19 000 00
*date 1902-2001
_ 1902_2001 20 000 00
*date 69.12.31
_ 19691231 21 000 00
*date 1969.12.30
_ 19691230 22 000 00
*date 1969.12
_ 196912 23 000 00
*date 1969
_ 1969 24 000 00
*date 70.01.01
_ 19700101 25 000 00
*date 70.01
_ 197001 26 000 00
*date 70.02
_ 197002 27 000 00
*date 70
_ 1970 28 000 00
*date 99.12.31
_ 19991231 29 000 00
*date 1970-2000
_ 1970_2000 30 000 00
*date 1970.01.02
_ 19700102 31 000 00
*date 1970.03
_ 197003 32 000 00
*date 1970.04
_ 197004 33 000 00
*date 1971 ; regression test 1.4.13 bug with comment after year
_ 1971 34 000 00
*date 1970-2001
_ 1970_2001 35 000 00
*date 1975.01.31
_ 19750131 36 000 00
*date 1975.02.28
_ 19750228 37 000 00
*date 1975.03.31
_ 19750331 38 000 00
*date 1975.04.30
_ 19750430 39 000 00
*date 1975.05.31
_ 19750531 40 000 00
*date 1975.06.30
_ 19750630 41 000 00
*date 1975.07.31
_ 19750731 42 000 00
*date 1975.08.31
_ 19750831 43 000 00
*date 1975.09.30
_ 19750930 44 000 00
*date 1975.10.31
_ 19751031 45 000 00
*date 1975.11.30
_ 19751130 46 000 00
*date 1975.12.31
_ 19751231 47 000 00
*date 1976.01.31
_ 19760131 48 000 00
*date 1976.02.29
_ 19760229 49 000 00
*date 1976.03.31
_ 19760331 50 000 00
*date 1976.04.30
_ 19760430 51 000 00
*date 1976.05.31
_ 19760531 52 000 00
*date 1976.06.30
_ 19760630 53 000 00
*date 1976.07.31
_ 19760731 54 000 00
*date 1976.08.31
_ 19760831 55 000 00
*date 1976.09.30
_ 19760930 56 000 00
*date 1976.10.31
_ 19761031 57 000 00
*date 1976.11.30
_ 19761130 58 000 00
*date 1976.12.31
_ 19761231 59 000 00
*date 2000.01.31
_ 20000131 60 000 00
*date 2000.02.29
_ 20000229 61 000 00
*date 2000.03.31
_ 20000331 62 000 00
*date 2000.04.30
_ 20000430 63 000 00
*date 2000.05.31
_ 20000531 64 000 00
*date 2000.06.30
_ 20000630 65 000 00
*date 2000.07.31
_ 20000731 66 000 00
*date 2000.08.31
_ 20000831 67 000 00
*date 2000.09.30
_ 20000930 68 000 00
*date 2000.10.31
_ 20001031 69 000 00
*date 2000.11.30
_ 20001130 70 000 00
*date 2000.12.31
_ 20001231 71 000 00
; These should generate one "date in future" warning each (until 2078!)
*date 1900-2078
_ 1900_2078 72 000 00
*date 2078.12.31
_ 20781231 73 000 00
*date 2078.06
_ 207806 74 000 00
*date 2078
_ 2078 75 000 00
*date 2016.08.25-2078.12.31 ; long survey trip!
_ 20160825_20781231 76 000 00
; And this should generate two
*date 2078.12.01-2078.12.31
_ 20781201_20781231 77 000 00
; Tests around new ISO date format
*date 1911-13
_ 1911_13 78 000 00
*date 1911-12
_ 1911_12 79 000 00
*end
