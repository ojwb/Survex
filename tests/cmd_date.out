./cmd_date.svx:5:7: warning: Assuming 2 digit year is 1900
 *date 00.01.01
       ^~
./cmd_date.svx:7:7: warning: Assuming 2 digit year is 1900
 *date 00.01
       ^~
./cmd_date.svx:9:7: warning: Assuming 2 digit year is 1900
 *date 00.02
       ^~
./cmd_date.svx:11:7: warning: Assuming 2 digit year is 1900
 *date 00
       ^~
./cmd_date.svx:25:7: warning: Assuming 2 digit year is 1902
 *date 02.01.01
       ^~
./cmd_date.svx:27:7: warning: Assuming 2 digit year is 1902
 *date 02.01
       ^~
./cmd_date.svx:29:7: warning: Assuming 2 digit year is 1902
 *date 02.02
       ^~
./cmd_date.svx:31:7: warning: Assuming 2 digit year is 1902
 *date 02
       ^~
./cmd_date.svx:45:7: warning: Assuming 2 digit year is 1969
 *date 69.12.31
       ^~
./cmd_date.svx:53:7: warning: Assuming 2 digit year is 1970
 *date 70.01.01
       ^~
./cmd_date.svx:55:7: warning: Assuming 2 digit year is 1970
 *date 70.01
       ^~
./cmd_date.svx:57:7: warning: Assuming 2 digit year is 1970
 *date 70.02
       ^~
./cmd_date.svx:59:7: warning: Assuming 2 digit year is 1970
 *date 70
       ^~
./cmd_date.svx:61:7: warning: Assuming 2 digit year is 1999
 *date 99.12.31
       ^~
./cmd_date.svx:148:12: warning: Date is in the future!
 *date 1900-2078
            ^~~~
./cmd_date.svx:150:7: warning: Date is in the future!
 *date 2078.12.31
       ^~~~~~~~~~
./cmd_date.svx:152:7: warning: Date is in the future!
 *date 2078.06
       ^~~~~~~
./cmd_date.svx:154:7: warning: Date is in the future!
 *date 2078
       ^~~~
./cmd_date.svx:156:18: warning: Date is in the future!
 *date 2016.08.25-2078.12.31 ; long survey trip!
                  ^~~~~~~~~~
./cmd_date.svx:159:7: warning: Date is in the future!
 *date 2078.12.01-2078.12.31
       ^~~~~~~~~~
./cmd_date.svx:159:18: warning: Date is in the future!
 *date 2078.12.01-2078.12.31
                  ^~~~~~~~~~
./cmd_date.svx:162:12: warning: Assuming 2 digit year is 1913
 *date 1911-13
            ^~
./cmd_date.svx:164:7: warning: Interpreting as an ISO-format date - use "*date surveyed 1911-12" to suppress this warning, or "*date 1911 1912" if you wanted a date range
 *date 1911-12
       ^~~~~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 80 survey stations, joined by 79 legs.
There are 0 loops.
Total length of survey legs = 3160.00m (3160.00m adjusted)
Total plan length of survey legs = 3160.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from 1911_12 at 0.00m to 1911_12 at 0.00m)
North-South range = 79.00m (from 1911_12 at 79.00m to _ at 0.00m)
East-West range = 0.00m (from 1911_12 at 0.00m to 1911_12 at 0.00m)
  79 1-nodes.
   1 79-node.
There were 23 warning(s).
